src/main/groovy/nebula/plugin/info/scm/UnknownScmProvider.groovy

```diff
@@ -17,11 +17,19 @@
 package nebula.plugin.info.scm
 
 import org.gradle.api.Project
+import org.gradle.api.provider.ProviderFactory
+
+import javax.inject.Inject
 
 class UnknownScmProvider extends AbstractScmProvider {
 
     public static final String LOCAL = 'LOCAL'
 
+    @Inject
```

Comment:
The injection is added for every plugin, but not for the other providers. Looks like this should use the passed in `providerFactory`?

Reply:
oh this is a leftover, wil lremove it! thanks

Reply:
addressed it in https://github.com/nebula-plugins/gradle-info-plugin/pull/66/commits/1d7dbf6921c8e75bc02ef6552b117ccfbc967534